opensrp-chw/src/nacp/assets/json.form/hivst_mobilization_session.json

```diff
@@ -114,18 +114,18 @@
         "openmrs_entity": "concept",
         "openmrs_entity_id": "female_condoms_issued",
         "type": "edit_text",
-        "hint": "Female condoms issued",
+        "hint": "Idadi ya kondomu za kike zilizotolewa",
         "v_numeric": {
           "value": "true",
-          "err": "Please enter a valid number"
+          "err": "Tafadhali ingiza numbari sahihi"
         },
         "v_min": {
           "value": "0",
-          "err": "The number must be equal or greater than 0"
+          "err": "Tarakimu ulioingiza inatakiwa iwe sawa au kubuwa kuliko 0"
         },
         "v_required": {
           "value": true,
-          "err": "Please enter the number"
+          "err": "Tafadhali ingiza nambar"
         }
       },
       {
```

Comment:
This needs to be in opensrp-chw/src/nacp/assets/json.form-sw/hivst_mobilization_session.json